submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/exceptions/InvalidDateFormatException.kt

```diff
@@ -0,0 +1,5 @@
+package ac.uk.ebi.biostd.submission.exceptions
+
+class InvalidDateFormatException(
+    date: String
+) : RuntimeException("Invalid date format provided for date $date. Expected format is YYYY-MM-DD")
```

Comment:
it looks confusing for me, maybe

`Provided date `$date` could not be parsed. Expected format is YYYY-MM-DD`